SPECS/lua/lua.spec

```diff
@@ -224,6 +216,10 @@ popd
 %{_libdir}/*.a
 
 %changelog
+* Tue May 09 2023 Bala <balakumaran.kannan@microsoft.com> - 5.4.4-1
+* Upgrade to version 5.4.4 to fix CVE-2021-44964
+* Removed patches that are already part of new version
```

Comment:
Can you change * to - in these changelog entries

Reply:
I saw this warning in the toolchain build:
error: bad date in %changelog: Removed patches that are already part of new version